therapy/etl/chembl.py

```diff
@@ -220,42 +220,46 @@ def _load_alias(self, record, batch):
         record['aliases'] = record['aliases'].split("||")
 
         # Remove duplicates (case-insensitive)
-        record['aliases'] = \
-            list(set({a.casefold(): a for a in
+        aliases = \
+            list(set({t.casefold(): t for t in
```

Comment:
Does this need to convert to a list? You might be able to keep it as a set unless you're calling .append() somewhere else.

Also, since you aren't using this for the final identity item (on line 238, it looks like you're pulling from the original `record` object rather than your transformed list) you could probably just keep this as a set of casefolded (ie non-duplicate) alias names and then you wouldn't have to call `lower()` on each one again on line 232.